mods/bones/init.lua

```diff
@@ -186,20 +186,42 @@ minetest.register_on_dieplayer(function(player)
 		bones_mode = "bones"
 	end
 
+	local enable_bones_logging = minetest.settings:get_bool("enable_bones_logging") ~= false
+	local enable_bones_chat_msg = minetest.settings:get_bool("enable_bones_chat_msg") == true
+	local player_name = player:get_player_name()
+	local pos = vector.round(player:get_pos())
+
 	-- return if keep inventory set or in creative mode
 	if bones_mode == "keep" or (creative and creative.is_enabled_for
-			and creative.is_enabled_for(player:get_player_name())) then
+			and creative.is_enabled_for(player_name)) then
+		if enable_bones_logging then
+			minetest.log("action", "[bones] " .. player_name ..
+				"'s bones do not remain since player has creative -- " ..
+				"died at " .. minetest.pos_to_string(pos))
```

Comment:
I find "bones do not remain" very awkward. It's also better to use "because" instead of "since". 
Perhaps: "player_name died at pos, but bones do not appear because player_name has creative."
This is also fixes the awkward way pos is tagged on the end right now.

These points (especially use of "since") apply to the other messages as well.

Reply:
I agree, 'bones do not remain' is bad English, and 'because' is better than 'since'.
Maybe 'bones are not placed' or something.

Reply:
Random suggestion:
(player) died at (pos). Bones were not placed because player has creative privilege